src/chia_log/handlers/condition_checkers/time_since_last_farm_event.py

```diff
@@ -44,7 +44,11 @@ def check(self, obj: HarvesterActivityMessage) -> Optional[Event]:
         elif seconds_since_last > self._info_threshold:
             # This threshold seems to be surpassed multiple times per day
             # on the current network. So it only generates an INFO log.
-            logging.info(f"Last farming event was {seconds_since_last} seconds ago. This is unusual.")
+            logging.info(
+                f"Last farming event was {seconds_since_last} seconds ago."
+                " Usually every 9-10 seconds. No reason to worry if it happens "
```

Comment:
I would prefer to add this to the wiki and keep the info here clean and concise.

Reply:
I'd agree for notification texts, but this is part of the actual process logs so we can afford more verbosity, I think. 🤔 I'd like that at least to some degree users can tell directly from the log message what they're seeing, not always having to refer to the wiki page.

Reply:
Either we do it this way or we hide this log on DEBUG level. Because many users will see this and won't see the wiki page.